testing/SendLog/SendLog.py

```diff
@@ -0,0 +1,120 @@
+## Hrishi Hiraskar
+## 23 September 2016
+
+import gevent
+import time
+from gevent import monkey
+from gevent.pywsgi import WSGIServer
+from flask import Flask, request, Response, render_template
+
+monkey.patch_all()
+
+app = Flask(__name__)
+
+# Delay time to look for new line (in s)
+LOOK_DELAY = 0.1	
+# States of the line
+INITIALIZATION = 0
+ENDING = 1
+DATA = 2
+NOLINE = -1
+# List to store figure IDs
+figure_list = []
+
+class line_and_state:
```

Comment:
Way to go! This is what I was expecting!

I feel that the polling mechanism could be better - but we will come back to that, if it does not work out well.

Overall, I'm extremely happy with your work here. We will proceed to plotting next. 

In the past, I have tried using Plotly - But it didn't work out that well.

So my opinion is to use, Highcharts with dynamic real time updates. Here are a couple of examples.

[1] http://www.highcharts.com/demo/dynamic-update
[2] http://www.highcharts.com/demo/synchronized-charts

Let me know your thoughts!